.dockerignore

```diff
@@ -1,2 +1,3 @@
-.git
-
+*
+!requirements
+!package.json
```

Comment:
this change (not sending repo files + node_modules + bower_components + whatever else to docker daemon) made a big difference in build time